core/src/main/java/org/apache/iceberg/util/DateTimeUtil.java

```diff
@@ -86,6 +83,10 @@ public static long microsFromTimestamptz(OffsetDateTime dateTime) {
   }
 
   public static String formatTimestampMillis(long millis) {
-    return DATE_FORMAT.format(LocalDateTime.ofInstant(Instant.ofEpochMilli(millis), ZoneOffset.UTC));
+    return Instant.ofEpochMilli(millis)
+        .atOffset(ZoneOffset.UTC)
```

Comment:
`Instant.ofEpochMilli(...).toString()` already formats in UTC, why bother with `.atOffset(UTC)`?

Reply:
you're right. updated